06-reactjs/views/main-view.js

```diff
@@ -0,0 +1,52 @@
+var React = require('react');
+var MovieStore = require('../store/movieStore');
+var MovieConstants = require('../store/movieConstants');
+var MovieTableView = require('../views/movie-table-view');
+var Router = require('react-router');
+var RouteHandler = Router.RouteHandler;
+
+var MainView = React.createClass({
+
+    getInitialState: function () {
+        return{
+            movies: MovieStore.getMovies(),
+            selected: MovieStore.getSelected()
+        }
+    },
+
+    //Listen for changes
+    componentDidMount: function () {
+        MovieStore.addChangeListener(MovieConstants.CHANGE_EVENT, this.updateMovies);
+    },
+
+    //Unbind change listener
+    componentWillUnmount: function () {
+        MovieStore.removeChangeListener(MovieConstants.CHANGE_EVENT, this.updateMovies);
+    },
+
+    render: function () {
+        return (
+            <div className="row">
+                <section className="text-center">
+                    <h1>Welcome to the amazing world of react</h1>
+                </section>
+                <div className="row">
+                    <MovieTableView movies={this.state.movies} />
+                    <RouteHandler />
+                </div>
+            </div>
+        )
+    },
+
+    updateMovies: function () {
+        this.setState(
+            {
```

Comment:
this should be one line up and the closing tag one line after like:

({
...
})